aperturedb/ParallelLoader.py

```diff
@@ -2,117 +2,23 @@
 import time
```

Comment:
This class seems redundant as *Loader translates to essentially parallel queries .
Is there a use of parallel loader which you foresee ?

We should delete this and derive the loaders from parallelquery.

Reply:
there is common functionality for all the loaders (as of now, just print_stats). Otherwise, we would have to implement a specific print_stat function for every loader.